Dockerfile

```diff
@@ -1,33 +1,6 @@
-FROM debian:stretch-slim
+FROM scratch
```

Comment:
I think this'll have the same (or worse) issue as the Alpine image. The built courier binaries need to link to glibc. If you want to take this approach, I would suggest the glibc tag of the [BusyBox image](https://hub.docker.com/r/library/busybox/). You can then add users [as you would with Alpine Linux](https://github.com/praekeltfoundation/docker-django-bootstrap/blob/develop/alpine/Dockerfile#L16-L17) as Alpine uses BusyBox.

Reply:
Thanks @JayH5, I think I'll leave it as is for now and keep this open, if we decide to go this route later.